.bit/responses/4.5-Week 4 Step 5.md

```diff
@@ -29,6 +29,8 @@ Week 4 Step 5 ⬤⬤⬤⬤⬤◯◯ | 🕐 Estimated completion: 5-20 minutes
 
 > 💡 **Tip:** It's a good idea to name your tags with `id`, since you'll then be able to use `document.getElementById` for modifying and getting values.
 
+> 🚨 **NOTE:** The CounselorBot will be checking the code to see if you use the [CataaS API](http://cataas.com/) endpoint. However, if the API is down, you may use the [substitute version](https://bit-cat.azurewebsites.net/cat/says/serverless) for testing purposes. Make sure to change it back when you submit your code!
```

Comment:
for this one, why are we asking them to change it back before they submit the code? because if the bot is going to call the code on the broken cataas API, won't it error out as well?

Reply:
@ganning127 great question! in cypress, there wasn't really a way to check if an image of a cat was produced, so when I wrote the test, I just checked if the src attribute of the image was the cataas endpoint: https://github.com/bitprj/Intro-To-Serverless/blob/main/.bit/tests/cypress/integration/4.5.spec.js

Reply:
in the cypress test, would there be a way to say it should include `https://cataas.com/cat/says/cat` OR it should include `https://bit-cat.azurewebsites.net/cat/says/serverless`? I'm trying to look through their docs for something like that: https://docs.cypress.io/api/commands/should#Function